doc/man7/EVP_SIGNATURE-ML-DSA.pod

```diff
@@ -81,6 +81,11 @@ Note that the message encoding steps from
 L<FIPS 204|https://csrc.nist.gov/pubs/fips/204/final> Algorithm 2 step 10 and
 Algorithm 3 step 5 are omitted when this setting is 1.
 
+See L<EVP_MD-EXTERNAL-MU-ML-DSA(7)> for more information on generating an
+external mu value.
+
+The "context-string" is ignored if this value is set to 1.
```

Comment:
Actually, any non-zero value is treated the same as `1`.

Reply:
Yes. I don't think this matters